### PR TITLE
A recursive function in JavaScript that prints all elements of a multi-dimensional array.

### PR DESCRIPTION
**Pull request by SGopinath89**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [ ] This pull request is all my own work -- I have not plagiarized.
- [ ] I know that pull requests will not be merged if they fail the automated tests.
- [ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [ ] All new JavaScript files are placed inside an existing directory.
- [ ] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [ ] All new algorithms have a URL in their comments that points to Wikipedia or another similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Recursive/MultiDimensionalArrayPrint.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  for (let i = 0; i < arr.length; i++) {
+    if (Array.isArray(arr[i])) {
+      // Recursive case: if the element is an array, go deeper
+      printElements(arr[i])
+    } else {
+      // Base case: if the element is not an array, print it
+      console.log(arr[i])
+    }
+  }
+}
```
Comment (anandfresh) [inline Recursive/MultiDimensionalArrayPrint.js:11]: Always use `return` in the function and console the function execution .

**Review by anandfresh — CHANGES_REQUESTED** [review]

File: Recursive/MultiDimensionalArrayPrint.js (hunk 1 above)
Comment (SGopinath89) [reply Recursive/MultiDimensionalArrayPrint.js:11]: It is the function used to print the element, why should I use "Return"?

**Comment by raklaptudirm** [comment]
IMO this algorithm is too simplistic to be worth adding to the repo.

**Comment by appgurueu** [comment]
> IMO this algorithm is too simplistic to be worth adding to the repo.

I concur.

**Pull request closed without merging**